packages/api/src/controllers/webhooksController.ts

```diff
@@ -115,6 +126,59 @@ export default async function webhookController(fastify: FastifyInstance) {
     },
   );
 
+  fastify.withTypeProvider<TypeBoxTypeProvider>().post(
+    "/amazonses",
+    {
+      schema: {
+        description: "Used to consume amazonses notification events.",
+        tags: ["Webhooks"],
+        body: AmazonSNSEvent,
+      },
+      onRequest: (req, _, done) => {
+        // eslint-disable-next-line no-param-reassign
+        req.headers["content-type"] = "application/json";
+        done();
+      },
+    },
+    async (request, reply) => {
+      logger().debug({ body: request.body }, "Received AmazonSES event.");
+
+      // Validate the signature
+      const valid = await validSNSSignature(request.body);
+
+      if (valid.isErr()) {
+        logger().error("Invalid signature for AmazonSES webhook.", valid.error);
+        return reply.status(401).send({ message: "Invalid signature" });
+      }
+
+      switch (request.body.Type) {
+        case AmazonSNSEventTypes.SubscriptionConfirmation:
+        case AmazonSNSEventTypes.UnsubscribeConfirmation:
```

Comment:
Subscriptions are handled internally, within dittofeed. In the case of other email service providers, like sendgrid, we ignore subscription change events, because we disable the subscription functionality within sendgrid and delegate it to Dittofeed. We should likely do the same for SES. Does that make sense?

Reply:
It does, but this is about the notification service (unrelated to SES) confirming the webhook is alive.  When the destination is added in the AWS console it sends a "SubscriptionConfirmation" to the webhook.  The webhook has to perform a GET on the "SubscribeUrl" property that was sent with the confirmation in order for the destination to become active and start receiving events.

Reply:
Ah ok, thank you for the clarification! Can you write a short comment?